src/drc/api/viewsets.py

```diff
@@ -208,6 +219,48 @@ def perform_destroy(self, instance):
 
         super().perform_destroy(instance.canonical)
 
+    def destroy_document_version(self, document: EnkelvoudigInformatieObject, versie: int) -> None:
```

Comment:
```suggestion
    def perform_destroy_single_version(self, document: EnkelvoudigInformatieObject, versie: int) -> None:
```
maybe? to keep the method name similar to `perform_destroy`